sqitch/deploy/instances.sql

```diff
@@ -10,7 +10,8 @@ CREATE TABLE cloudbrain.instances (
 	state         TEXT  NOT NULL,
 	ip_address    TEXT,
 	ssh_key       TEXT,
-	upstream_id   TEXT
+	upstream_id   TEXT,
+	error_reason  TEXT
```

Comment:
This kinda works since Cloud Brain isn't deployed yet, but once we are at the point where we can't drop the database and recreate it, this should be a new migration and not added to the old one.